vllm/model_executor/guided_decoding/guidance_logits_processors.py

```diff
@@ -0,0 +1,195 @@
+# SPDX-License-Identifier: Apache-2.0
+import json
+import os
+from typing import Any, List, Type, Union
+
+import llguidance  # type: ignore[import-untyped]
```

Comment:
We can probably remove this type: ignore since https://github.com/guidance-ai/llguidance/pull/139